src/pages/part8.jsx

```diff
@@ -57,13 +57,18 @@ const Part8 = () => {
       </Typography>
       <div>
       {/* TODO: (2) Make a typography with variant="h4" */}
+      <Typography variant="h4" align="center">
         <h4>Sign Up For Our Mailing List</h4>
+      </Typography>
       {/* TODO: (3) Change the p tags to Typography with variant="body1" */}
+      <Typography variant="body1" align="center">
```

Comment:
Every `<p>` should be replaced by a Typography. Generally, don't put tags inside Typography.